configmap/parse.go

```diff
@@ -144,6 +146,33 @@ func AsQuantity(key string, target **resource.Quantity) ParseFunc {
 	}
 }
 
+// AsNamespacedName parses the value at key as a types.NamespacedName into the target, if it exists
+func AsNamespacedName(key string, target *types.NamespacedName) ParseFunc {
+	return func(data map[string]string) error {
+		raw, ok := data[key]
+		if !ok {
+			return nil
+		}
+
+		v := strings.Split(raw, string(types.Separator))
```

Comment:
we can do SplitN(3) — to avoid doing more work if there are say 30 `/` :)

Reply:
done!